### PR TITLE
SDK-6520 Fix ubuntu json parsing

### PR DESCRIPTION
**Pull request by YevhenServetnykSD**

### CODE REVIEW

File: plugins/websocket/include/websocket/Output.h [hunk 1]
```diff
@@ -73,7 +73,9 @@ class PLUGIN_API Output final : public ResultsOutput
 	void operator()(const std::string& metadata, const dto::ImageView*) noexcept final
 	{
 		using namespace boost::json;
-		m_client.sendResult(std::move(parse(string_view{metadata.data(), metadata.size()}).as_object()));
+		parser json_parser;
```
Comment (liviuivan-neurala) [inline plugins/websocket/include/websocket/Output.h:76]: We should use camel case for our variables; i.e. `jsonParser`.

File: plugins/websocket/include/websocket/Output.h [hunk 2]
```diff
@@ -73,7 +73,9 @@ class PLUGIN_API Output final : public ResultsOutput
...
+		json_parser.write(reinterpret_cast<const char*>(metadata.data()), metadata.size());
```
Comment (liviuivan-neurala) [inline plugins/websocket/include/websocket/Output.h:77]: I don't see a need for a cast here. Isn't `const char*` already the returned type?

File: plugins/websocket/src/Client.cpp [hunk 3]
```diff
@@ -76,8 +76,10 @@ Client::metadata() noexcept
 	try
 	{
 		using namespace boost::json;
-		const value jsonValue{
-		 parse(string_view{reinterpret_cast<const char*>(buffer.data()), buffer.size()})};
+		parser jsonParser;
+		jsonParser.write(string_view{reinterpret_cast<const char*>(buffer.data()), buffer.size()});
+		value jsonValue = jsonParser.release();
+
 		const object& jsonObject{jsonValue.as_object()};
```
Comment (liviuivan-neurala) [inline plugins/websocket/src/Client.cpp:83]: Here I wouldn't use `string_view` if there is an overload that accepts pointer + size as direct parameters. It seems possible to contract the 2nd and 3rd instructions somehow but I don't understand the reference/value semantics these functions have so the final call is yours.

**Review by liviuivan-neurala — APPROVED** [review]

**Comment by YevhenServetnykSD** [comment]
http://172.16.102.144:8080/job/nightly_plugins/20/

**Pull request merged**